src/index.html

```diff
@@ -1,13 +1,453 @@
+                <div class="testimonial__photo-container">
+                  <img
+                    src="./images/Met/10.png"
+                    alt="photo-10"
+                    class="testimonial__photo"
+                  >
+                </div>
+                <h3 class="testimonial__topic">
+                  Virtual group tours
+                </h3>
+                <p class="testimonial__description">
+                  Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title page__section-title--bottom">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-3 grid__item-desktop--1-6 grid--contact">
+              <div class="grid__item grid__item-desktop--1-3">
+                <div class="contact-us__fifth-avenue">
+                  <h3 class="contact-us__title">
+                    The Met Fifth Avenue
+                  </h3>
+                  <p class="contact-us__info">Phone</p>
+                  <a href="tel: +1 212-535-7710" class="contact-us__info-number">
+                    +1 212-535-7710
+                  </a>
+                  <p class="contact-us__info">Address</p>
+                  <a
+                    href="https://goo.gl/maps/rFPu1xkgwvEvhQur5"
+                    class="contact-us__info-address"
+                    target="_blank"
+                  >
+                    1000 Fifth Avenue
+                    <br>
+                    New York, NY 10028
+                  </a>
+                </div>
+              </div>
+              <div class="grid__item grid__item-desktop--4-6">
+                <div class="contact-us__cloisters">
+                  <h3 class="contact-us__title">
+                    The Met Cloisters
+                  </h3>
+                  <p class="contact-us__info">Phone</p>
+                  <a href="tel: +1 212-923-3700" class="contact-us__info-number">
+                    +1 212-923-3700
+                  </a>
+                  <p class="contact-us__info">Address</p>
+                  <a
+                    href="https://goo.gl/maps/xgp4TWvJ8zGnnGqr9"
+                    class="contact-us__info-address"
+                    target="_blank"
+                  >
+                    99 Margaret Corbin Drive
+                    <br>
+                    Fort Tryon Park
+                    <br>
+                      New York, NY 10040
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-6 grid__item-desktop--8-12">
+              <form action="get">
```

Comment:
Checklist 15th item, that's what I see when I submit a form
![image](https://user-images.githubusercontent.com/37161315/170655363-e2462510-64ad-43cb-b5ab-8bfa5740c035.png)